src/unix/os390.c

```diff
@@ -666,13 +685,21 @@ static int os390_message_queue_handler(uv__os390_epoll* ep) {
   __a2e_l(msg.__rfim_utok, sizeof(msg.__rfim_utok));
   handle = *(uv_fs_event_t**)(msg.__rfim_utok);
   assert(handle != NULL);
+
+  assert((handle->flags & UV_HANDLE_CLOSED) == 0);
+  if (uv__is_closing(handle)) {
+    uv__handle_stop(handle);
+    uv__make_close_pending((uv_handle_t*) handle);
+    return 0;
+  }
```

Comment:
I think the change in uv__fs_event_stop to remove the uv__handle_stop needs it must detect that case now here?
```suggestion
  } else if (handle->path == NULL) {
      /* _RFIS_UNREG returned EALREADY */
      uv__handle_stop(handle);
      return 0;
  }
```